pkg/daemon/metrics.go

```diff
@@ -367,6 +385,17 @@ func UpdateInterfaceRoleMetrics(process string, iface string, role ptpPortRole)
 		"process": process, "node": NodeName, "iface": iface}).Set(float64(role))
 }
 
+func UpdateProcessStatusMetrics(process string, cfgName string, status int64) {
```

Comment:
Can you define "up" and "down" constants that you pass to UpdateProcessStatusMetrics as the status instead of an int64?

Reply:
Done